challenges.yaml

```diff
@@ -192,12 +192,12 @@
       Count the number of files in the current
       working directory. Print the number of
       files as a single integer.
-    example: ls | wc -l
-    example_fail: echo 2
```

Comment:
I updated it and ran it locally. Why does the build fail? You just use the output in `expected_output` isn't it?